axelrod/strategies/averagecopier.py

```diff
@@ -18,17 +21,14 @@ class AverageCopier(Player):
 
     @staticmethod
     def strategy(opponent):
-        """Randomly picks a strategy (not affected by history)."""
         if len(opponent.history) == 0:
+            # Randomly picks a strategy (not affected by history).
             return random_choice(0.5)
         p = opponent.cooperations // len(opponent.history)
-        rnd_num = random.random()
-        if rnd_num < p:
-            return 'C'
-        return 'D'
+        return random_choice(p)
 
 
-class NiceAverageCopier(Player):
+class NiceAverageCopier(AverageC):
```

Comment:
Have opened https://github.com/risicle/Axelrod/pull/1 (which looks terribly messy because of all the brutal commits we panickly wrote to try and fix read the docs but importantly fixes that tiny typo).